example/locals.tf

```diff
@@ -83,13 +83,13 @@ locals {
   }
 
   tanium_install = {
```

Comment:
Would it not be better these as variables to the module instead and then set in internal-compute?